WordPress/Classes/Extensions/NSURLCache+Helpers.swift

```diff
@@ -1,49 +0,0 @@
-import Foundation
-
-
-/// Encapsulates all of the NSURLCache Helpers
-///
-extension URLCache {
-    /// Private Constants
-    ///
-    fileprivate struct Constants {
-        static let statusCodeOK     = 200
-        static let httpVersion      = "1.1"
-        static let expirationKey    = "Expires"
-        static let expirationTime   = TimeInterval(3600)
-        static let contentTypeKey   = "Content-Type"
-        static let contentTypePNG   = "image/png"
-    }
-
-
-    /// Updates the cache contents for a given request, and stores the specified image.
-    ///
-    /// - Parameters:
-    ///     - image: the image that should be stored on top of (whatever) is currently cached
-    ///     - request: the request that should produce the given image
-    ///
-    @objc func cacheImage(_ image: UIImage, forRequest request: URLRequest) {
-        guard let URL = request.url,
-            let responseData = image.pngData() else {
-            return
-        }
-
-        // First: Prepare the Response Headers
-        let headerFields = [
-            Constants.contentTypeKey: Constants.contentTypePNG,
-            Constants.expirationKey: Date(timeIntervalSinceNow: Constants.expirationTime).toStringAsRFC1123()
```

Comment:
`toStringAsRFC1123` is also unused, but it's in WordPressShared, so I can't tell for sure if anyone else is using it.